src/app/products/loan-products/loan-product-stepper/loan-product-terms-step/loan-product-terms-step.component.ts

```diff
@@ -306,7 +306,7 @@ export class LoanProductTermsStepComponent implements OnInit, OnChanges {
   }
 
   get loanProductTerms() {
-    return this.loanProductTermsForm.value;
+    return this.loanProductTermsForm.getRawValue();
```

Comment:
How does it fix the problem when the "isZeroInterest" checkbox is ticked, the interestRate field had no value?

Reply:
The problem exists If you set a control as disabled, independently of the value on this control the method `value` ignores and does not returns the value of the controls disabled. The other method `getRowValue()` send all the values of the controls including the disabled ones